src/main/java/ListOfTasks.java

```diff
@@ -0,0 +1,130 @@
+import java.util.ArrayList;
+
+public class ListOfTasks {
+    private static int count;
+    private ArrayList<Task> xs;
+    private Ui ui;
+
+    public ListOfTasks() {
+        //this.xs = new Task[100];
+        ui = new Ui();
+        this.xs = new ArrayList<>();
+        this.count = 0;
+    }
+
+    public ArrayList<Task> getList() {
+        return xs;
+    }
+
+    public void includeAdditionalTask(Task x) {
```

Comment:
Do you think this method with this long name is necessary to implement your logic?